index.html

```diff
@@ -853,66 +868,227 @@ <h2 id="BaseAudioContext">
               be played to the destination, and then allows the system to
               release its claim on audio hardware. This is generally useful
               when the application knows it will not need the
-              <a>BaseAudioContext</a> for some time, and wishes to let the
-              audio hardware power down. The promise resolves when the frame
+              <a>BaseAudioContext</a> for some time, and wishes to temporarily
+              <a>release system resource</a> associated with te
+              <a>BaseAudioContext</a>. The promise resolves when the frame
               buffer is empty (has been handed off to the hardware), or
               immediately (with no other effect) if the context is already
-              suspended. The promise is rejected if the context has been
-              closed.
+              <code>"uspended"</code>. The promise is rejected if the context
```

Comment:
Typo: "uspended" -> "suspended".  To match other changes we probably want to remove the quotes.